### PR TITLE
React Tooltip Component

### PR DESCRIPTION
**Pull request by Colinh84**
Test examples are located under the Misc > TooltipExamples menu.

### CODE REVIEW

**Comment by Colinh84** [comment]
Rebased to updated master branch to get rid of all the other dev branch changes.

File: components/Tooltip/Tooltip.jsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            return child.props.children
+          })}
+      </div>
+    )
+    let ttClasses = 'Tooltip tooltip-hide' + ' ' + tooltipTheme
+    if (typeof this.props.className !== 'undefined') {
+      ttClasses += ' ' + this.props.className
+    }
+    return (
+      <div className={ttClasses}>
```
Comment (vikasrohit) [inline components/Tooltip/Tooltip.jsx]: Can you please use classnames module for this? You can see other components for reference. Many of the existing components use this module.

**Comment by Colinh84** [comment]
@vikasrohit Added classnames module.

**Comment by vikasrohit** [comment]
Thanks @Colinh84

**Comment by vikasrohit** [comment]
Merging. Will post final fix list soon.

@nlitwin Can you please have a look at code once and let us know if there is anything that can be improved?

**Pull request merged**